docs/database/database-spans.md

```diff
@@ -478,6 +480,43 @@ Semantic conventions for individual database systems or specialized instrumentat
 MAY specify a different `db.query.summary` format as long as produced summary remains
 relatively short and its cardinality remains low comparing to the `db.query.text`.
 
+## Propagating context to databases
+
+**Status**: [Development][DocumentStatus]
+
+Instrumentations SHOULD propagate the context information to the SQL queries following [sqlcommenter](https://google.github.io/sqlcommenter/spec/).
```

Comment:
I wanted to share a learning from my org. We've implemented a sqlcommenter-like approach for our instrumentations and we found it more beneficial to _prepend_ the sql comment rather than _append_ to the query text. This runs counter to the sqlcommenter spec, but we found that most databases truncate the query in views like pg_stat_activity, so prepending the attributes offered a better guarantee that they wouldn't be truncated

Reply:
@zacharycmontoya Thank you for pointing this out!

I think it makes sense to prepend the sql comment instead. I'll update this PR accordingly if there are no objections. cc @open-telemetry/semconv-db-approvers @XSAM

Reply:
Using prepend conflicts the spec of sql commenter.

@zacharycmontoya Could you elaborate more about this? How serious and often the truncate will happen to certain databases if we use append.

Reply:
Postgres [limits query text to 1024 characters by default](https://www.postgresql.org/docs/current/monitoring-stats.html#MONITORING-PG-STAT-ACTIVITY-VIEW), but you can increase it.

Reply:
Another idea is to make it flexible by allowing the implementation of propagation for a specific database to choose between `append` or `prepend`.

Reply:
Modify this PR to incorporate the idea.

Reply:
Modified this PR to incorporate the idea.